tests/installation/bootloader_uefi.pm

```diff
@@ -86,6 +86,11 @@ sub run {
     if (get_var('UEFI_HTTP_BOOT') || get_var('UEFI_HTTPS_BOOT')) {
         tianocore_http_boot;
     }
+
+    # Some aach64 JeOS jobs take too long to match the first grub2 needle.
+    # By pressing a random key, we stop the grub timeout
+    send_key 'backspace' if (is_jeos && is_aarch64);
```

Comment:
why not just create disable timeout during image creation ? I think with that approach you will still have cases when openQA will be too slow to press key in time ...

Reply:
The users need the timeout to be able to boot it headless, or at least boot it without user action.

Reply:
We can't. Images come like this. JeOS is not an image we create in openQA.

Reply:
ah ok than